### PR TITLE
MSEARCH-102: Add docs for supported queries.

### PR DESCRIPTION
**Pull request by bohdan-suprun**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,3 +88,163 @@ In order to configure connection to elasticsearch you have to provide following
...
+
+| Option                        | Type |Example                                    | Description                  |
+| :-----------------------------|:----:| :-----------------------------------------|:------------------------------|
+| `holdings.id`                 | term | `holdings.id=="1234567"`                  | Matches instances that have a holding with the id |
+| `holdings.permanentLocationId`| term | `holdings.permanentLocationId=="123765"`  | Matches instances that have holdings with given permanentLocationId |
+| `holdings.discoverySuppress`  | term | `holdings.discoverySuppress==true`        | Matches instances that have holdings suppressed/not suppressed from discovery |
+| `holdings.hrid`               | term | `holdings.hrid=="hr10*3"`                 | Matches instances that have a holding with given HRID |
+| `holdingTags`                 | term | `holdingTags=="important"`                | Matches instances that have holdings with given tags |
+| `holdingsFullCallNumbers`     | term | `holdingsFullCallNumbers="cn*434"`        | Matches instances that have holdings with given call number string (prefix + call number + suffix) |
```
Comment (pfilippovEPAM) [inline README.md:222]: electronic access for holdings and items are missing

File: README.md [hunk 2]
```diff
@@ -88,3 +88,163 @@ In order to configure connection to elasticsearch you have to provide following
...
+### Recreating Elasticsearch index
+
+Sometimes we need to recreate Elasticsearch index, for example when a breaking change introduced to ES index
+structure (mapping).
+
+Here are the steps how to do it:
+1. Remove ES index:
+```http
+   DELETE [ES_HOST]/instance_[tenant_name]
+```
```
Comment (hjiebsco) [inline README.md:103]: Wondering if we should add mod-search API to abstract this?

File: README.md [hunk 3]
```diff
@@ -88,3 +88,163 @@ In order to configure connection to elasticsearch you have to provide following
...
+
+There is no end-to-end monitoring implemented yet, however it is possible to monitor it partially. In order to check
+how many records published to Kafka topic use inventory API:
+```http
+GET [OKAPI_URL]/instance-storage/reindex/[reindex job id]
+```
+_reindex job id_ - id returned by `/search/index/inventory/reindex` endpoint.
+
+In order to estimate total records that actually added to the index, you can send a "match all" search query and check
+`totalRecords`, e.g. `GET /search/instances?query=id="*"`.
```
Comment (hjiebsco) [inline README.md:134]: Or query ES directly to see how many docs are indexed.

**Review by hjiebsco — APPROVED** [review]
LGTM. Please add what @pfilippovEPAM pointed out.

File: README.md (hunk 1 above)
Comment (bohdan-suprun) [reply README.md]: Thanks, added.

File: README.md (hunk 2 above)
Comment (bohdan-suprun) [reply README.md:103]: Make sense. I'm thinking about introducing a `recreateIndex=true` parameter for reindex API. I'll create a story for it.

Comment (bohdan-suprun) [reply README.md:103]: Here is the story - https://issues.folio.org/browse/MSEARCH-121

**Pull request merged**